### PR TITLE
feat!: Replaced Nunito Sans and Regular with Apercu; also added Suisse

### PR DESCRIPTION
**Pull request by JoshuaKGoldberg**
### PR Checklist

- [x] Related to Abstract designs: https://www.figma.com/file/DIWmrLJ4iaesJH0mwvsiqV/Identity-Phase-1?node-id=1954%3A0
- [x] Related to JIRA ticket: GM-68
- [x] I have run this code to verify it works
- [x] This PR includes tests for the code change

### Description

Totally and completely removes `Nunito Sans` and `Regular Bold`, and replaces them with `Apercu`. Also adds in `Suisse` as an opt-in.

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@1.9.6-alpha.f71bab.0<br/>@codecademy/gamut-styles@4.2.6-alpha.f71bab.0<br/>@codecademy/gamut@13.0.1-alpha.f71bab.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f3b09ec8012bd017a05e893--gamut-preview.netlify.app>https://5f3b09ec8012bd017a05e893--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f3b09ec8012bd017a05e893>Deploy Logs</a></p>

File: packages/gamut-styles/core/_fonts.scss [hunk 1]
```diff
@@ -1,31 +1,61 @@
 @font-face {
-  font-family: "Nunito Sans";
```
Comment (JoshuaKGoldberg) [inline packages/gamut-styles/core/_fonts.scss:2]: Note: the old fonts will almost certainly need to be added back in for the monolith & this PR renamed. I'm just testing for now.

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@1.9.6-alpha.f25439.0<br/>@codecademy/gamut-styles@4.2.6-alpha.f25439.0<br/>@codecademy/gamut@13.0.1-alpha.f25439.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f3c16fefc39b879553d7943--gamut-preview.netlify.app>https://5f3c16fefc39b879553d7943--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f3c16fefc39b879553d7943>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@2.0.1-alpha.fdc23c.0<br/>@codecademy/gamut-styles@5.0.1-alpha.fdc23c.0<br/>@codecademy/gamut@14.0.1-alpha.fdc23c.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f3d820661c717465b219219--gamut-preview.netlify.app>https://5f3d820661c717465b219219--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f3d820661c717465b219219>Deploy Logs</a></p>

File: packages/gamut-styles/utils/variables/_typography.scss [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
+$font-family-base: "Apercu", -apple-system, BlinkMacSystemFont, "Segoe UI",
   "Roboto", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
   sans-serif;
 
 $font-base: $font-family-base;
 
-$font-family-headings: "Regular Patch", "Regular Bold", "Nunito Sans",
-  -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Ubuntu", "Cantarell",
-  "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif;
+$font-family-headings: "Apercu", -apple-system, BlinkMacSystemFont, "Segoe UI",
```
Comment (jakemhiller) [inline packages/gamut-styles/utils/variables/_typography.scss:9]: Let's add a font family variable for `suisse` just so people can use it easily. What do you think of `$font-family-accent`? Maybe the design team has a term for what role suisse has.

**Comment by JoshuaKGoldberg** [comment]
Closing in favor of #935.

**Pull request closed without merging**